src/frontend/src/utils/change-requests.utils.ts

```diff
@@ -0,0 +1,9 @@
+import { ChangeRequest, StandardChangeRequest } from 'shared';
+
+export const canReviewChangeRequest = (cr: ChangeRequest, selected: number) => {

```

Comment:
this is a pretty weird helper method, I would just keep it in the file where it's being used

Reply:
I might've misunderstood but you said to make this a helper method

Reply:
ohhh I see, sorry should've been more specific. The second half, not the selected stuff, should be a helper method and you can name is "hasProposedChanges" or smthn